sentinelsat/sentinel.py

```diff
@@ -192,9 +191,11 @@ def load_query(self, query, start_row=0, products=[]):
 
         # repeat query until all results have been loaded
         if total_results > self.page_size + start_row - 1:
-            self.load_query(query, start_row=(start_row + self.page_size), products=output)
+            output += self.load_query(query, start_row=(start_row + self.page_size))
+            return output
```

Comment:
You can safely remove this line (`return output`). Other than that, everything looks good. :)